app/scripts/views/form.js

```diff
@@ -166,7 +166,14 @@ define(function (require, exports, module) {
       // applied, no submit handler is fired, and the form validation does not
       // take place.
       if (this.isFormEnabled()) {
-        this.$('button[type=submit]').addClass('disabled');
+        if (this.isInExperimentGroup && this.isInExperimentGroup('disabledButtonState', 'treatment')) {
+          this.logEvent('exp.dbs.treatment.render');
+        } else if (this.isInExperimentGroup && this.isInExperimentGroup('disabledButtonState', 'control')) {
+          this.logEvent('exp.dbs.control.render');
```

Comment:
In contrast, with `forceExperimentGroup=control`, this event is only logged once per form enable->disable sequence.